src/test/java/net/starlark/java/eval/testdata/bench_int.star

```diff
@@ -0,0 +1,25 @@
+_11 = 1 << 11        # 32-bit integer
+_33 = 1 << 33        # 64-bit integer
+_ml = (1 << 63) - 1  # maxlong
+_66 = 1 << 66        # BigInteger
+
+def bench_int_add_32(b):
```

Comment:
We should probably change the --filter flag so that it operates on the string "int:bench_add_32", where the part before the colon comes from the file name, and then we can avoid repeating "int" in the name of the function (def bench_add32). I'll do that in a follow-up.